### DISCUSSION TITLE
Add a documentation for the API !

### DISCUSSION THREAD

**Discussion by RoyaLProg** [TheNewEconomy/TNE-Bukkit#498] [Ideas]
I think its vital for other developpers wo would love to dev with the plugin to get a correct documention !

Even just class and method declaration would be great !

Like that we would be able to do things and know what i spossible with the API

**Comment by creatorfromhell** [comment]
> I think its vital for other developpers wo would love to dev with the plugin to get a correct documention !
> 
> Even just class and method declaration would be great !
> 
> Like that we would be able to do things and know what i spossible with the API

Beyond the already detailed javadocs? https://github.com/TheNewEconomy/TNE-Bukkit/blob/master/TNE/src/net/tnemc/core/common/api/TNEAPI.java

**Comment by RoyaLProg** [comment]
Looks like i made a basic new java dev mistake x)

**Reply by creatorfromhell** [reply]
> Looks like i made a basic new java dev mistake x)

No worries about it. Never hurts to ask :P
